lib/pf/Switch/Aruba/ArubaSwitchNG.def

```diff
@@ -0,0 +1,24 @@
+# - better handling of disconnection than ArubaSwitch.pm
+# - don't want a getIfIndexByNasPortId function for these switches
+description = Aruba Switch NG
+radiusDisconnect = coaOrDisconnect
+
+acceptVlan = <<EOT
+Tunnel-Medium-Type = 6
+Tunnel-Type = 13
+Tunnel-Private-Group-ID = $vlan
+EOT
+
+acceptRole = <<EOT
+Filter-ID = $role
+EOT
+
+disconnect = <<EOT
+Calling-Station-Id = ${macToEUI48($mac)}
+NAS-IP-Address = $disconnectIp
+EOT
+
+coa = <<EOT
+Calling-Station-Id = ${macToEUI48($mac)}
+NAS-IP-Address = $disconnectIp
```

Comment:
I don't think its necessary to add this as its implicit from what I know

Unless that overrides the default behavior?